src/output-json.c

```diff
@@ -152,6 +152,86 @@ json_t *JsonAddStringN(const char *string, size_t size)
     return SCJsonString(tmpbuf);
 }
 
+void EveFileInfo(JsonBuilder *jb, const File *ff, const bool stored)
+{
+    size_t filename_size = ff->name_len * 2 + 1;
+    char filename_string[filename_size];
+    BytesToStringBuffer(ff->name, ff->name_len, filename_string, filename_size);
```

Comment:
can we skip this step and us a `jb_set_string_from_bytes`-style func?

Reply:
Yes; will adjust.

Reply:
I've taken this PR in for merge in #5110, so can you address this in a follow up PR?